README.md

```diff
@@ -31,6 +31,7 @@ It does:
 | Name           | Default Value | Description                        |
 | -------------- | ------------- | -----------------------------------|
 | `default['mondoo']['registration_token']` | `change_me` | Mondoo Registration Token that is used to retrieve client credentials
+| `default['mondoo']['api_proxy']` | `` | Mondoo Proxy setting for Mondoo API traffic
```

Comment:
```suggestion
| `default['mondoo']['api_proxy']` | `` | Proxy server URL for communication with Mondoo Platform
```

Reply:
It's not for this PR, but we should really get rid of this .tap usage. That's not going to be something most Chef users get.